src/pages/CDP.js

```diff
@@ -114,6 +123,53 @@ const CdpViewCard = ({ title, rows, isAction }) => {
   );
 };
 
+const CdpViewHistory = ({ title, rows }) => {
+  return (
+    <Box>
+      <CardTitle title={title} />
+      <Card px="m" py="s" my="s">
+        <Table width="100%" variant="normal">
+          <thead>
+            <tr>
+              <th>{lang.table.type}</th>
+              <th>{lang.table.activity}</th>
+              <th>{lang.table.time}</th>
+              <th>{lang.table.sender_id}</th>
+              <th>{lang.table.tx_hash}</th>
+            </tr>
+          </thead>
+          <tbody>
+            {rows.map(
+              (
+                [collateralType, actionMsg, dateOfAction, senderId, txHash],
+                i
+              ) => (
+                <tr key={i}>
+                  <td>{collateralType}</td>
+                  <td>{actionMsg}</td>
+                  <td>{dateOfAction}</td>
+                  <td>{senderId}</td>
+                  <td>{txHash}</td>
+                </tr>
+              )
+            )}
+          </tbody>
+        </Table>
+      </Card>
+    </Box>
+  );
+};
+
+const AddrLink = ({ address }) => (
+  <Link
+    fontWeight="400"
+    href={`https://etherscan.io/address/${address}`}
```

Comment:
we probably wanna use this util here https://github.com/makerdao/mcd-cdp-portal/blob/6f7a21dde181b6ecef66a7d2e5e6aaced7e74261/src/utils/ethereum.js#L54

Reply:
exactly what I was looking for!